manifests/config.pp

```diff
@@ -25,6 +25,13 @@
     content => template('foreman/database.yml.erb'),
   }
 
+  file { '/etc/systemd/system/httpd.service/10-longer_timeout.conf':
```

Comment:
For systemd to pick this up you need to do a ```systemctl daemon-reload``` but in other another module we've started to use ```systemd::dropin```.

Reply:
I now also see that it should be ```httpd.service.d```, at least on RH systems. For better compatibility I'd use ```${::apache::service_name}.service.d``` since on Debian it's ```apache2.service``` and RH users can set up their system to use the SCL ```httpd24-httpd``` service name. The ```systemd::dropin``` will also take care of ensuring the parent directory exists.